hardware/hardware.mk

```diff
@@ -288,6 +288,7 @@ $(BSG_MACHINE_PATH)/bsg_bladerunner_pkg.v: $(BSG_MACHINE_PATH)/bsg_bladerunner_c
 	@echo "parameter bsg_machine_core_dmem_words_gp = 1024;" >> $@
 	@echo "parameter bsg_machine_core_icache_entries_gp = 1024;" >> $@
 	@echo "parameter bsg_machine_core_icache_tag_width_gp = 12;" >> $@
+	@echo "parameter bsg_machine_core_icache_line_words_gp = 4;" >> $@
```

Comment:
so is `bsg_machine_core_icache_line_words_gp` = `icache_block_size_in_words_p`?

Reply:
Yes?

Reply:
we had a discussion about naming offline